plugins/GSdx/GSRenderer.cpp

```diff
@@ -293,7 +306,7 @@ bool GSRenderer::Merge(int field)
 			m_dev->ShadeBoost();
 		}
 
-		if (m_shaderfx)
+		if(m_shaderfx)
 		{
```

Comment:
> Turtleli would've felt sad if he saw those inconsistent placement of if() checks :/

Sounds like turtleli died or something.

> He'd better not check any of the other files. :|

FTFY.

Generally it's best to keep cleanup/consistency changes low when making a commit that affects functionality so it's easy to tell what the functionality change is. My commits aren't the best example of this though.